scripts/db_migrations/v2.20/upload_models_s3.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import uuid
+from pymongo import MongoClient
+import boto3
+import re
+
+##### Connect to AWS S3 #####
+s3 = boto3.client(
+  's3',
+  aws_access_key_id='AKIAJIJNXOYFX5B6SHXA',
+  aws_secret_access_key='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',
+  region_name='eu-west-1',
+)
+bucket_name = '3d-models-test'
+
+mongoURL = sys.argv[1]
+mongoPort = sys.argv[2]
+userName = sys.argv[3]
+password = sys.argv[4]
+connString = "mongodb://"+ userName + ":" + password +"@"+mongoURL + ":" + mongoPort + "/"
+
+##### Enable dry run to not commit to the database #####
+dry_run = True
+
+##### Connect to the Database #####
+db = MongoClient(connString)
+for database in db.database_names():
+  if database != "admin" and database != "local":
+    db = MongoClient(connString)[database]
+    print("--database:" + database)
+
+##### Get a model ID #####
+    for setting in db.settings.find():
+      def upload(col_prefix):
```

Comment:
don't define a function in the middle of the code. This way of coding is very hard to follow.
In fact for this situation you don't even need an extra function, just do:
````python
for col_prefix in [".history",".stash.json_mpc", ".stash.unity3d" ]:
   #...
````